lib/bolt/config/src/ns.rs

```diff
@@ -543,6 +543,8 @@ pub struct Rivet {
 	pub telemetry: Telemetry,
 	#[serde(default)]
 	pub access: RivetAccess,
+	#[serde(default = "trueable")]
+	pub admin_login: bool,
```

Comment:
Nit: should be moved under smoething like `rivet.login.enable_admin` or something like that.